web/src/styles/scss/print.scss

```diff
@@ -80,6 +79,15 @@
     padding: 0 !important;
     border: none !important;
   }
+
+  .public-DraftEditor-content {
+    border: 0;
+  }
+
+  .rte-textarea-l .public-DraftEditor-content,
+  .textarea-l {
+    min-height: 0;
```

Comment:
Was there a reason this had a `min-height` at all?

Reply:
No idea.  Maybe it's to mimic the textareas?  Without a `min-height`, it's just as tall as the content, so it kind of looks like a one-line textfield.